_resources/port1.0/group/fortran-1.0.tcl

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+#
+# This PortGroup provides support for building Fortran packages.
+#
+# Usage:
+# PortGroup         fortran 1.0
+
+PortGroup           compiler_blacklist_versions 1.0
+PortGroup           compilers 1.0
+PortGroup           github 1.0
+
+default categories  "fortran"
+
+depends_build-append \
+                    port:fpm
+
+compiler.blacklist-append \
+                    *gcc-4.*
+
+global os.platform os.arch
+platform darwin powerpc {
+    compiler.blacklist-append *clang*
+}
+
+compilers.choose    fc f90
+compilers.setup     require_fortran -g95
+
+use_configure       no
+
+pre-build {
+    # Make gfortran symbolic link
+    xinstall -m 0755 -d ${workpath}/bin
+    ln -s ${configure.fc} ${workpath}/bin/gfortran
+
+    build.env-append \
+        PATH=$env(PATH):${workpath}/bin \
+        FC=${configure.fc}
+}
+
+build.cmd           ${prefix}/bin/fpm install
+build.target
+
+options             fortran.profile
+global              prefix
+default profile     release
+build.cmd-append    --prefix="${workpath}${prefix}" --profile="${profile}"
+
+destroot {
+    move ${workpath}${prefix} ${destroot}${prefix}
```

Comment:
One lingering concern is whether this `move` will overwrite files already in place.  Pre-destroot, MacPorts installs certain core files into the destroot prefix before running the port-specific `destroot` block or command. So it could be that all gets blown away by this `move` command.

Reply:
Ok, so what gonna be the correct way? We do not know exactly which files gonna be installed, since different ports may install different components. Use foreach with copy and if fileexists?

Reply:
`fs-traverse` could be an option here

Reply:
> `fs-traverse` could be an option here

Possibly I have implemented it :)